SUPPORT.md

```diff
@@ -0,0 +1,44 @@
+# Support Guidelines
+
+We strive to create clear guidelines on communication to the Zarf team to provide a good community experience.
+
+
+## Standard Process
+All questions, and code issues should be a [Github Issue](https://github.com/defenseunicorns/zarf/issues/new/choose) that follows the issue template. 
+
+Following the templates provides the Zarf community a foundation of understanding to be able to assist quickly.
+After an issue is made, this issue can be brought into other chanels such as the [Kubernetes Slack #Zarf](https://zarf.dev/slack) channel or the [bi-weekly Zarf Community Meeting](zarf.dev/???).
```

Comment:
@salaxander  - When the biweekly community meetings get set up, I can update this on the zarf.dev/support page?